rkt/api_service.go

```diff
@@ -51,7 +51,7 @@ var (
 
 func init() {
 	cmdRkt.AddCommand(cmdAPIService)
-	cmdAPIService.Flags().StringVar(&flagAPIServiceListenClientURL, "--listen-client-url", common.APIServiceListenClientURL, "address to listen on client API requests")
```

Comment:
@joshix Good found! In fact both should work. The quotes are added by the cobra package. I updated the USAGE, now both have the quotes.
But this makes me double think again, `localhost:15441` is not a url in fact, as it does not contain the `scheme` part.